introduction.md

```diff
@@ -1,4 +1,6 @@
 Yasuo the Unforgiven is based on a legendary samurai.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?